src/js/leafletMap.js

```diff
@@ -0,0 +1,162 @@
+/* global L, fluid */
+
+"use strict";
+
+var hortis = fluid.registerNamespace("hortis");
```

Comment:
This`lifeletMap` uses a different namespace. Can you sync up namespaces used in this repo? Thanks.

Reply:
The contents of this file have been borrowed from a different project and will hopefully be factored back out again into a separate library once it can be established what can best be shared